### ISSUE TITLE
Use the same Zig version for darwin

### ISSUE THREAD

**Issue by Jacalz** [fyne-io/fyne-cross-images#46]
The macOS images (darwin ) use a different version of Zig but does not motivate exactly whey. If this is fixed with newer versions, we should try to move to using the same one for all targets to avoid code complexity.

See https://github.com/fyne-io/fyne-cross-images/blob/25ee7237006fb6e9336ebc8ca6d476d931f6afa1/darwin/Dockerfile#L9

Potentially related to https://github.com/fyne-io/fyne-cross-images/issues/21.

**Comment by Jacalz** [comment]
> I understand the version of Zig to be very fragile and the specifics were picked for toolchains that worked in testing. That was a while ago though.

Indeed. I'm trying to make sure that all of the specific (and poorly documented) workarounds are documented somewhere. This doesn't necessarily mean that we need to remove them right now but we need to be aware of why they exist as well as how and when they can be removed.

Like you say, this was a while ago and without an issue to track it and with poor code documentation, it would unfortunately just have rotted away in the codebase.

**Comment by lucor** [comment]
Unfortunately we cannot move to a newer version yet. See https://github.com/fyne-io/fyne-cross/pull/276

**Comment by Jacalz** [comment]
If something isn't working upstream then we should open a bug there otherwise we can't be certain that it is ever going to be fixed.

**Comment by sdassow** [comment]
This should be fixed now with e0fbc9d6f028f75d2ce5d44e11e411ee74223b09.

**Comment by sdassow** [comment]
Closing this as new images are released using the same zig version.
